lib/rubocop/cop/metrics/class_length.rb

```diff
@@ -37,17 +39,12 @@ def on_class(node)
         end
 
         def on_casgn(node)
-          class_definition?(node) do
-            check_code_length(node)
-          end
+          _scope, _name, def_node = *node
```

Comment:
It seems like an instance of `BlockNode`.

```suggestion
          _scope, _name, block_node = *node
```